assets/js/app/Instance.js

```diff
@@ -264,19 +262,23 @@ LocusZoom.Instance.prototype.mapTo = function(chr, start, end){
         this.remap_promises.push(this.panels[id].reMap());
     }
 
-    // When all finished update download SVG link
-    Q.all(this.remap_promises).then(function(){
-        this.controls.setBase64SVG();
-    }.bind(this), function(error){
-        console.log(error);
-        this.curtain.drop(error);
-    }.bind(this));
+    Q.all(this.remap_promises)
+        .then(this.mapToComplete)
+        .catch(function(error){
+            console.log(error);
+            this.curtain.drop(error);
+        }.bind(this));
 
     return this;
     
 };
 
+// Logic to fire when an instance has finished mapping to a new region
+LocusZoom.Instance.prototype.mapToComplete = function(){
```

Comment:
There are conceivable situations where some logic should fire only after a remap is complete. The main use case was for the export branch (#27, still in a holding pattern) that needed to update the serialized/encoded SVG string in the download link any time a remap happened. Another example with tooltips, potentially, would be to redraw any tooltips for previously selected data points. However it's not clear that such things would need to happen only at the end of a remap, so I just pulled out that stubbed method entirely on this branch. It, or something like it, can be reapproached when it's actually needed for something.